source/2019-09-25-ember-3-13-released.md

```diff
@@ -19,9 +19,9 @@ So why is this a preview and not the final Octane release?
 
 A big part of Octane is about delivering an excellent developer experience, whether you are a beginner or a power user. While these APIs are stable, not all Octane features are compatible with popular addons yet. We are also still updating documentation, and important tools like the Ember Inspector do not yet work with some Octane features. Octane is not complete until these supporting pieces are in place.
 
-Then, when Ember 3.14 arrives, Octane will be the recommended way to use Ember and new apps will have Octane's optional features enabled by default. The guides and tutorials will show Octane examples, and codemods will be available to help users migrate to Octane. Feel free to take a look at the [current draft](https://emberjs.com/editions/octane) of these learning resources, which are deployed to a temporary URL.
+Then, when Ember 3.14 arrives, Octane will be the recommended way to use Ember and new apps will have Octane's optional features enabled by default. The guides and tutorials will show Octane examples, and codemods will be available to help users migrate to Octane. Feel free to take a look at the [current draft](https://octane-guides-preview.emberjs.com/release/) of these learning resources, which are deployed to a temporary URL.
```

Comment:
How about moving the link, that way we have them both? 

...users migrate to [Octane](https://emberjs.com/editions/octane).

Reply:
I'm not sure if I understand the suggestion, can you clarify what you mean?

I think the intent here is a specific call-to-action, pointing the readers to read the guides/tutorial, so I would like to link directly to the relevant thing from the the text.